common/services.yml

```diff
@@ -11,17 +11,20 @@ x-logging: &default-logging
 
 services:
   maindb:
-    image: mysql:latest
+    image: postgres:latest
     environment:
-      - MYSQL_ROOT_PASSWORD=root
+      - POSTGRES_USER=backend
```

Comment:
If you end up reopening this, the user credentials are set in the 000_init_db_and_users.sql file and the user configured here should be the "root" user